services/syllabus/models.py

```diff
@@ -0,0 +1,89 @@
+##########################################################################
+# Name:     Syllabus
+# Purpose: File contains User details of all the members in the Org
+#
+# Author:     Sai Nikhita Guduru
+#
+# Created:   07/07/2019
+# Copyright:   (c) Hoder Solutions Pvt Ltd 2018 - Present
+# Licence:   <your licence>
+##########################################################################
+from main import db
+from json import loads
+
+
+class Syllabus(db.Model):
+    __tablename__ = "syllabus"
+    __table_args__ = (
+        db.UniqueConstraint('term', 'class_no', 'subject', name='unique_component_commit'),
+    )
+    # unique identifier for a user, todo: need to have uuid or mobile as unique identifier
+    id = db.Column(db.Integer, primary_key=True)
+    # Term of syllabus
+    term = db.Column(db.String(200), nullable=False)
```

Comment:
I am not sure about the term, can you get this approved by Vijay, like what a term means and is it different from the academic year.